roles/swift-object/templates/etc/swift/object-expirer.conf

```diff
@@ -0,0 +1,19 @@
+[DEFAULT]
+log_name = object-expirer
+log_udp_host = 127.0.0.1
+log_udp_port = 514
+
+[object-expirer]
+
+[pipeline:main]
+pipeline = catch_errors cache proxy-server
+
+[app:proxy-server]
+use = egg:swift#proxy
+
+[filter:cache]
+use = egg:swift#memcache
+memcache_servers = 127.0.0.1:11211
```

Comment:
No - modeled this after proxy memcache configuration to start with. 

Planning to add patch where network memcache is used for both configurations